pkgs/applications/editors/neovim/wrapper.nix

```
@@ -72,31 +75,37 @@ let
   in
   assert withPython2 -> throw "Python2 support has been removed from the neovim wrapper, please remove withPython2 and python2Env.";
 
-  symlinkJoin {
+  stdenv.mkDerivation (finalAttrs: {
       name = "neovim-${lib.getVersion neovim}${extraName}";
+
+      __structuredAttrs = true;
+      unpackPhase = ":";
```

this might be a little better?

```suggestion
      dontUnpack = true;
```